### DISCUSSION TITLE
Import from other files

### DISCUSSION THREAD

**Discussion by Ekiben542** [community/community#128739] [Programming Help] [answered]
### Select Topic Area

Product Feedback

### Body

Hello, I am trying to import from a python file in another directory to a python file in a different directory, but for some reason I can't import it. How can I do this?

**Comment by davevad93** [comment, marked as answer]
Hi @Ekiben542 ,

I guess you're talking about importing a `module` from folder1 into a `py` file in folder2 or viceversa, so let's imagine the structure is as follows: 

project
├── folder1
│   ├── __init__.py
│   └── main.py
├── folder2
│   ├── __init__.py
│   └── app.py
└── __init__.py

For instance, if you want to import `app.py` from `folder2` into `main.py` in `folder1` you have to do this:

```python
# "project" as the name of main folder and "folder2" as the name of the folder containing the module to import
from project.folder2 import app 
```
It's mandatory to have a `init.py` file in the root of your project's folder, otherwise the `import` doesn't work. Anyway, here's a very useful resource about importing files in `Python`:

- [Relative imports](https://stackoverflow.com/questions/14132789/relative-imports-for-the-billionth-time)

**Reply by ebndev** [reply]
Hi @davevad93, thanks for still being active 🌟

@Ekiben542 if the response above helped, please feel encouraged to mark it as the answer. If not, let us know if you have further questions 😄
